great_expectations/render/renderer/content_block/table_content_block.py

```diff
@@ -45,7 +45,10 @@ def expect_column_unique_value_count_to_be_between(cls, ge_object):
     @classmethod
     def expect_column_proportion_of_unique_values_to_be_between(cls, ge_object):
         observed_value = ge_object["result"]["observed_value"]
-        return [["Distinct (%)", "%.1f%%" % (100*observed_value)]]
```

Comment:
Be prepared to deal with None. Not doing this results in failure to render

Reply:
👍 

We're going to have to make similar changes all over the rendering engine.